unlock-app/src/components/interface/FinishSignup.tsx

```diff
@@ -16,6 +16,24 @@ interface Props {
 interface State {
   password: string
   passwordConfirmation: string
+  errors: string[]
+}
+
+export const validatePassword = (
+  password: string,
+  passwordConfirmation: string
+) => {
+  let errors: string[] = []
+
+  if (password.length < 1) {
```

Comment:
maybe choose a number a bit longer, and probably put that in constants.js?

Reply:
This is only there to ensure it's not empty. Ultimately there will be an additional check for password complexity, ideally including a meter indicating whether a password is weak, medium, or strong.